pkg/scaffold/api.go

```diff
@@ -74,11 +72,11 @@ func (api *API) Validate() error {
 
 func (api *API) setDefaults() error {
 	if api.project == nil {
-		p, err := LoadProjectFile("PROJECT")
+		p, err := util.LoadProjectFile("PROJECT")
```

Comment:
👍 I liked a lot. It really needs to be centralized in util. 
WDYT about rename `util` for `utils`

Reply:
I used `util` because it already existed. I have plans to move everything related to the Project config file, including loading, saving, etc to a separate package.

Reply:
Just for reference, this has been done in #1325